src/init/starship.fish

```diff
@@ -5,14 +5,14 @@ function fish_prompt
         case '*'
             set keymap insert
     end
-    set -l exit_code $status
+    set exit_code $status
```

Comment:
If I understand this correctly `-l` would prevent this from setting a previously defined variable with the same name? It might be better to use `-l` everywhere in that case.

Reply:
Yes, but we should use a more specific name like `starship_exit_code` from the beginning.

Reply:
I would prefer `-l` everywhere, but prefixing everything with `starship_` is also fine.